packages/devtools_app/lib/src/app.dart

```diff
@@ -105,6 +106,14 @@ class DevToolsAppState extends State<DevToolsApp> with AutoDisposeMixin {
   void initState() {
     super.initState();
 
+    // TODO(https://github.com/flutter/devtools/issues/6018): Once
+    // https://github.com/flutter/flutter/issues/129692 is fixed, disable the
+    // browser's native context menu on secondary-click, and instead use the
+    // menu provided by Flutter:
+    if (kIsWeb) {
+      unawaited(BrowserContextMenu.enableContextMenu());
```

Comment:
do we really want to do this at the app-wide level? Will this break any flutter context menu actions we use elsewhere? Perhaps we could enable this menu more local to the console / debugger.

Reply:
I'm pretty sure this is a no-op (the context menu is already globally enabled) - what we really want to do is disable it, but currently are blocked by https://github.com/flutter/flutter/issues/129692). I can switch this to disabling and just comment it out.

Reply:
What about disabling the context menu closer to the source of where we have actually implemented our own context menus? Like in this example how the browser context menu is disabled and reenabled within the lifecycle of this widget: https://github.com/flutter/flutter/blob/e12167d98c44ad1977934399757a4351821f4f95/examples/api/lib/material/context_menu/context_menu_controller.0.dart#L34-L50

Reply:
I considered doing that, though I'm not sure where the best place would be. Would we want it on the entire debugger panel (and if so, would it be weird that that one panel has different context menus then the rest of DevTools)? Or on each selectable text region (which could get us into a weird state, eg context menu is disabled when codeview inits its state, then when the variable tree inits its state, then the variable tree is removed and the context menu is enabled, even though we still want it for codeview)?

I think we probably want the Flutter context menus everywhere. This matches the behavior for macos, and so ideally means the experience during development matches the experience in release-mode.